.lintstagedrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "*": [
+        "npm ci",
+        "npx pepr format --validate-only",
+        "yamllint -c .yamllint --no-warnings"
+    ]
```

Comment:
Also would be neat to add https://github.com/codespell-project/codespell here. It should generate few/no false-positives because of how it works, but might be good to setup an ignore file anyways just in case?